python/py-pyaudio/Portfile

```diff
@@ -57,5 +57,6 @@ if {${name} ne ${subport}} {
     livecheck.type      none
 } else {
     livecheck.type      pypi
-    livecheck.name      ${my_name}
+    livecheck.url       [lindex ${master_sites} 0]
+    livecheck.regex     pyaudio-(\[0-9.\]+)${extract.suffix}
```

Comment:
What was the purpose of this change, given that we already have the `pypi` livecheck type?